dxr/templates/layout.html

```diff
@@ -49,20 +49,25 @@
                           </button>
                         </section>
                         <div class="sf-select-options sf-modal" aria-expanded="false">
-                          <ul class="selector-options" tabindex="-1">
+                          <table class="selector-options" tabindex="-1">
                             {% for filter in filters %}
-                              <li>
-                                <a href="javascript:void(0)" data-value="{{ filter.name }}:">
-                                  <span class="selector-option-label">
-                                    {{ filter.name }}
-                                  </span>
-                                  <span class="selector-option-description">
-                                    {{ filter.description }}
-                                  </span>
-                                </a>
-                              </li>
+                              <tr data-value="{{ filter.name }}:">
+                                <td class="selector-option-label">
+                                  {{ filter.name }}
+                                </td>
+                                <td class="selector-badges">
+                                  {% for text, color in filter.badges %}
+                                    {% if color %}
+                                      <span class="badge-label" style="background-color:{{ color }}">{{ text }}</span>
+                                    {% endif %}
```

Comment:
It does sort alphabetically, but first the non-language filters and then the language filters (hasattr 'lang').